apps/tlon-mobile/src/screens/UserBugReportScreen.tsx

```diff
@@ -0,0 +1,193 @@
+  const insets = useSafeAreaInsets();
+  const [state, setState] = useState<'initial' | 'sent'>('initial');
+
+  useEffect(() => {
+    const showEvent =
+      Platform.OS === 'ios' ? 'keyboardWillShow' : 'keyboardDidShow';
+    const hideEvent =
+      Platform.OS === 'ios' ? 'keyboardWillHide' : 'keyboardDidHide';
+
+    const keyboardDidShowListener = Keyboard.addListener(showEvent, () => {
+      setKeyboardVisible(true);
+    });
+    const keyboardDidHideListener = Keyboard.addListener(hideEvent, () => {
+      setKeyboardVisible(false);
+    });
+
+    return () => {
+      keyboardDidHideListener.remove();
+      keyboardDidShowListener.remove();
+    };
+  }, []);
+
+  const onGoBack = useCallback(() => {
+    props.navigation.goBack();
+  }, [props.navigation]);
+
+  const { control, handleSubmit, formState } = useForm({
+    defaultValues: {
+      additionalNotes: '',
+    },
+  });
+
+  const sendBugReport = useCallback(
+    (submission: { additionalNotes: string }) => {
+      console.log(`got additional notes`, submission.additionalNotes);
+      const reporter = new ErrorReporter(
+        'User manually submitted a bug report'
+      );
+      if (submission.additionalNotes) {
+        reporter.log(`User attached notes:`);
+        reporter.log(submission.additionalNotes);
+      }
+      reporter.report(null);
+      setState('sent');
+      setTimeout(() => {
+        if (isFocusedRef.current) {
+          onGoBack();
+        }
+      }, 3000);
+    },
+    [onGoBack]
+  );
+
+  return (
+    <KeyboardAvoidingView style={{ flex: 1 }}>
+      <GenericHeader goBack={state === 'initial' ? onGoBack : undefined} />
+      <YStack
+        marginTop="$m"
+        flex={1}
+        marginHorizontal="$2xl"
+        justifyContent={keyboardVisible ? 'unset' : 'space-between'}
+        onPress={() => Keyboard.dismiss()}
+      >
+        <YStack>
+          {!keyboardVisible && (
+            <YStack
+              backgroundColor="$secondaryBackground"
+              padding="$l"
+              alignItems="center"
+              borderRadius="$l"
+            >
+              <Circle marginBottom="$s">
+                <Icon type="Send" />
+              </Circle>
+              <SizableText fontSize="$xl">Send a bug report</SizableText>
+              <SizableText
+                marginHorizontal="$2xl"
+                marginTop="$l"
+                fontSize="$s"
+                lineHeight="$s"
+                color="$secondaryText"
+                textAlign="center"
+              >
+                If you experienced an issue, let us know! Sending reports helps
+                us improve the app for everyone.
+              </SizableText>
+            </YStack>
+          )}
+          <View paddingTop="$2xl">
+            {state === 'initial' ? (
+              <>
+                <SizableText
+                  marginHorizontal="$m"
+                  marginBottom="$l"
+                  fontSize="$s"
```

Comment:
`SizableText` only requires `size` prop set -- corresponding `fontSize` and `lineHeight` will be set automatically.

Reply:
so I thought! But it looked off to me, explicitly setting to `$s` does seem to have a visual change...

Reply:
Whoops, was surprised line height wasn't getting set. Forgot to use the right prop